pkg/kubectl/cmd/drain.go

```diff
@@ -17,11 +17,11 @@ limitations under the License.
 package cmd
 
 import (
+	"encoding/json"
```

Comment:
Please use `"k8s.io/apimachinery/pkg/util/json"`, even though it doesn't have any difference when you only do marshal.